experiment/resources/runner-startup-script-template.sh

```diff
@@ -28,19 +28,19 @@ do
   echo 'Error pulling image, retrying...'
 done{% endif %}
 
-docker run {% if local_experiment %}-v {{host_gcloud_config}}:/root/.config/gcloud {% endif %}\
---privileged --cpus=1 --rm \
+docker run --privileged --cpus=1 --rm \
 -e INSTANCE_NAME={{instance_name}} \
 -e FUZZER={{fuzzer}} \
 -e BENCHMARK={{benchmark}} \
 -e EXPERIMENT={{experiment}} \
 -e TRIAL_ID={{trial_id}} \
 -e MAX_TOTAL_TIME={{max_total_time}} \
```

Comment:
Are you giving the containers access to the filestores? If not why not?

Reply:
We give. Sorry not make all changes pushed when I create the PR.